specification/components/schemas/PatientSearch.yaml

```diff
@@ -0,0 +1,75 @@
+type: object
+properties:
+  resourceType:
+    type: string
+    description: FHIR resource type.
+    default: "Patient"
+    readOnly: true
+  id:
+    $ref: "NhsNumber.yaml"
+    readOnly: true
+  identifier:
+    $ref: "Identifier.yaml"
+  meta:
+    $ref: "Meta.yaml"
+  name:
+    type: array
+    description: | 
+      List of names associated with the patient.
```

Comment:
It is not all names, it is only usual, alias and preferred so old and maiden will not be included unless the historic flag is set.  Should that be reiterated?
System effective from/to only returned where history had been requested, are they excluded?

Reply:
It is actually only the `home` address on search and that is addressed in the line below:
`This will only be fully populated on a retrieval, only a the `home` address will be returned on a search.`

Reply:
The business effective dates are included

Reply:
I was talking about names not addresses.